lib/widgets/body/what_is_mission.dart

```diff
@@ -0,0 +1,88 @@
+import 'package:flutter/material.dart';
+import 'package:web/scroll_notifier.dart';
+import 'package:web/widgets/common/description_text.dart';
+import 'package:web/widgets/common/title_text.dart';
+import 'package:web/widgets/common/animated_widget.dart' as my;
+
+class WhatIsMission extends StatefulWidget {
```

Comment:
Can you check a background color? It's not rgba(0, 0, 0, 0.8).

expect: 
<img width="302" alt="image" src="https://user-images.githubusercontent.com/60090790/196123583-40838841-a1f7-40cb-9d0c-95393210daca.png">

now: 
<img width="302" alt="image" src="https://user-images.githubusercontent.com/60090790/196123902-fbe631f3-d94d-4300-b49a-f07bcb62fda3.png">

Reply:
Thanks for letting me know, I didn't notice it!! I changed to container to assign a color.